plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/ExchangeBean.java

```diff
@@ -91,6 +92,12 @@ protected void validatePositivity(BigDecimal... values) {
         }
     }
 
+    protected void validateDate(long date, long now) {
```

Comment:
Dont pass validation argument to validate function. This way validation logic is partially here and partially elsewhere. You can just do `if (date > now().toEpochMillis())`.

Maybe even better you can use java api and compare instants as `if(date.isAfter(now()))`